docparse/processing_options.mdx

```diff
@@ -19,15 +19,33 @@ All of the available options are listed below:
 * `extract_images`: If `true`, the partitioner crops each region identified as an image and attaches it to the associated `ImageElement`. This can later be fed into the `SummarizeImages` transform when used within Sycamore.
 * `selected_pages`: You can specify a page (like `[11]` ), a page range (like `[[25,30]]` ), or a combination of both (like ``[11, [25,30]]`` ) of your PDF to process. The first page of the PDF is `1`, not `0`.
 * `pages_per_call`: This is only available when using the Partition function in Sycamore. This option divides the processing of your document into batches of pages, and you specify the size of each batch (number of pages). This is useful when running OCR on large documents. 
+* `chunking_options`: A tree of options for specifying chunking behavoir. Chunking is only performed when this option is present, and the default options are chosen when chunking_options is specified as just `{}`. The full options are available in the [OpenAPI spec](../api-reference/endpoint/partition).
+    * `merging_strategy`: Strategy to use to combine and split chunks. Valid values are 'header_augmenter', 'greedy_text_element', and 'marked'.
+        * Behavior of `header_augmenter`:
```

Comment:
Remove `Behavior of` throughout. It is repetitive and unnecessary in my opinion.

Reply:
Ok, we can punt this for later.